src/components/registration-lite.js

```diff
@@ -127,6 +127,7 @@ const RegistrationLite = (
         clearWidgetState,
         requestedTicketTypes,
         allowPromoCodes,
+        showCompanyInput,
```

Comment:
please define the prop on props types here
https://github.com/OpenEventKit/summit-registration-lite/blob/2340fc2ecfd131a68e2f020ba3df851f335301ef/src/components/registration-lite.js#L424
and add a default value here https://github.com/OpenEventKit/summit-registration-lite/blob/2340fc2ecfd131a68e2f020ba3df851f335301ef/src/components/registration-lite.js#L411 ( true )